gdcdictionary/schemas/summary_clinical.yaml

```diff
@@ -2369,6 +2370,8 @@ properties:
       - "60 to 69"
       - "70 to 79"
       - "greater than 80"
+      - "16-64" # IDPH Vaccination dataset
+      - "+65" # IDPH Vaccination dataset
```

Comment:
could we use "16 to 64" and "greater than 65" for consistency with the other enum values?

Reply:
Yes, done.